packages/seacas/CMakeLists.txt

```diff
@@ -17,6 +17,30 @@ TRIBITS_ADD_OPTION_AND_DEFINE(
 
 set(${PROJECT_NAME}_C_Standard_DEFAULT c99)
 
+if (TPL_ENABLE_MPI)
+   if (NOT TPL_Netcdf_PARALLEL)  
+      ASSERT_DEFINED(TPL_Netcdf_INCLUDE_DIRS)
+      find_path(meta_path
+         NAMES "netcdf_meta.h"
+         HINTS ${TPL_Netcdf_INCLUDE_DIRS}
+         NO_DEFAULT_PATH)
+
+      if (meta_path)
+         # Search meta for NC_HAS_PARALLEL setting...
+         # Note that there is both NC_HAS_PARALLEL and NC_HAS_PARALLEL4, only want first...
+         file(STRINGS "${meta_path}/netcdf_meta.h" netcdf_par_string REGEX "NC_HAS_PARALLEL ")
+         string(REGEX MATCH "[01]" netcdf_par_val "${netcdf_par_string}")
+         if (netcdf_par_val EQUAL 1)
+            set(TPL_Netcdf_PARALLEL True CACHE BOOL "Is NetCDF library parallel enabled")
+         endif()
+      endif()
+      if (NOT TPL_Netcdf_PARALLEL)  
+         set(TPL_Netcdf_PARALLEL False CACHE BOOL "Is NetCDF library parallel enabled")
+      endif()
+   endif()
```

Comment:
I am not sure about this logic.  I think it could have bad behavior.  I think this fails if the variable `TPL_Netcdf_PARALLEL` is set to `FALSE` or `OFF` before it is called.  For example, if it is already set in the cache as `FALSE` or `OFF` before calling this code, then this code will **not** change the value of `TPL_Netcdf_PARALLEL`.

What are your assumptions about the nature of the variable `TPL_Netcdf_PARALLEL` before this code is called?

Reply:
This maybe should be `if (NOT DEFINED TPL_Netcdf_PARALLEL)` although there is also some argument that could be made that the value should always be set based on the setting in `netcdf_meta.h` since if the code includes that file, it may get conflicting values?

We can discuss on Skype if that works better

Reply:
> there is also some argument that could be made that the value should always be set based on the setting in netcdf_meta.h since if the code includes that file

Then the logic for setting `TPL_Netcdf_PARALLEL` should be in the file `FindTPLNetcdf.cmake` and not in this SEACAS file.  And we need to be careful about cache vs non-cache vars.

> We can discuss on Skype if that works better

A 5-min talk, I promise.

Reply:
As we discussed, please move this logic to the end if `FindTPLNetcdf.cmake` and change it to:

IF (NOT "${TPL_Netcdf_PARALLEL}" STREQUAL "")
   ...
   set(TPL_Netcdf_PARALLEL True CACHE INTERNAL "Is NetCDF library parallel enabled")
   ...
ENDIF()

You will also want to change the other `TPL_Netcdf_PARALLEL` set statement in `FindTPLNetcdf.cmake` type to `INTERNAL` instead of `BOOL` since the user should not be allowed to set it.